### ISSUE TITLE
Embedding of GSP ID and PV System ID

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/nowcasting_dataloader#21]
## Detailed Description
Let's experiment with giving our models:

* An embedding of the GSP ID (used in the query?  And for each row of GSP-level PV data if we use historical GSP-level PV?)
* An embedding of the PV System ID (could be used as pat of the position encoding for each row of PV data)

**Comment by JackKelly** [comment]
This implemented now, I think. Is that right, @jacobbieker?

**Comment by jacobbieker** [comment]
Yeah, we give it the ID, but we don't actually do anything with the ID. AS in, if a systems ID is 576, then the model gets 576 as the input, if that works?

**Comment by JackKelly** [comment]
Yeah... although it might be slightly better to first wrap the ID in an [embedding](https://pytorch.org/docs/stable/generated/torch.nn.Embedding.html) (which should "magically" cluster similar GSPs in vector space... and should help the model understand that GSP ID 100 isn't necessarily "similar" to ID 101, even though they have adjacent numerical IDs)

**Comment by jacobbieker** [comment]
Yeah, that sounds good!
